README.md

```diff
@@ -1 +1,25 @@
-# charmed-magma
\ No newline at end of file
+<a href="url"><img src="docs/images/magma-logo-purple.svg" width="300" ></a>
+
+**Magma** is an open source 4G and 5G core network. Use it to operate your own private mobile 
+network. Magma enables better connectivity by:
+
+- Allowing operators to offer cellular service without vendor lock-in with a modern, open source core network
+- Enabling operators to manage their networks more efficiently with more automation, less downtime, better predictability, and more agility to add new services and applications
+- Enabling federation between existing MNOs and new infrastructure providers for expanding rural infrastructure
+- Allowing operators who are constrained with licensed spectrum to add capacity and reach by using Wi-Fi and CBRS
+
+To learn more about Magma, visit the official [website](https://magmacore.org/).
+
+# Charmed Magma
+
+**Charmed Magma** is a distribution of Magma published by Canonical built to work with 
+[Juju](https://juju.is/). It is the easiest way to operate Magma. It provides the simplest setup, integration and 
+lifecycle management options.
+
+- **Works on all clouds**: Charmed Magma is not opinionated about your choice of cloud. It can also 
+run on your personnal computer.
+- **Integrations**: Charmed Magma offers seamless integration between each component and with 
+standard monitoring tools.
+- **Automation**: Charmed Magma automates the initial setup and most operations.
```

Comment:
```suggestion
- Allowing operators to offer cellular service without vendor lock-in with a modern, open source core network.
- Enabling operators to manage their networks more efficiently with more automation, less downtime, better predictability, and more agility to add new services and applications.
- Enabling federation between existing MNOs and new infrastructure providers for expanding rural infrastructure.
- Allowing operators who are constrained with licensed spectrum to add capacity and reach by using Wi-Fi and CBRS.

To learn more about Magma, visit the official [website](https://magmacore.org/).

# Charmed Magma

**Charmed Magma** is a distribution of Magma published by Canonical built to work with 
[Juju](https://juju.is/). It is the easiest way to operate Magma. It provides the simplest setup, integration and 
lifecycle management options.

- **Works on all clouds**: Charmed Magma is not opinionated about your choice of cloud. It can also 
run on your personnal computer.
- **Integrations**: Charmed Magma offers seamless integration between each component and with 
standard monitoring tools.
- **Automation**: Charmed Magma automates the initial setup and most operations.
```

Reply:
Fixed